### PR TITLE
Added blockwishlist module & Removed archived modules

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Added blockwishlist module & Removed archived modules
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | Install PrestaShop using this PR branch. The archived modules are now gone. The blockwishlist module version is v2.0.0 . Please verify the blockwishlist works as expected.
| Possible impacts? | N/A

Ping @marionf & @lolath-presta 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23617)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

**Review by matks — COMMENTED** [review]
I see in composer.lock you removed
```
"plugin-api-version": "1.1.0"
```

Is it intentional?

**Comment by Progi1984** [comment]
@matks Fixed

**Review by matthieu-rolland — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by matks** [comment]
We should write this in devdocs as an notable change in 1.7.8 (archived modules no longer bundled in the ZIP) 🤔

**Review by atomiix — DISMISSED** [review]

**Comment by matks** [comment]
I request QA validation as removing and adding modules in PS can have unexpexted consequences 😅

**Review by PierreRambaud — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @Progi1984,

After a fresh install, the `blockwishlist` module is not installed.
image
But I can find it on the Module Catalog page 
image

About archived modules:
1. sekeywords
2. statsequipment
3. statsorigin
4. statsvisits

They are not installed, they do not exist on the module manager page.
But I can find them on the Module Catalog page.
1. sekeywords
 image
2. statsequipment
image
3. statsorigin
image
4. statsvisits
image

As discussed with @marionf:
The `blockwishlist` module is a native pre-installed module, so after fresh install 178, this module should be installed & exists on the Module manager page

About native archived modules, they must be removed & they must be not displayed on the Module Catalog page.

Thank you!

**Comment by Progi1984** [comment]
I just removed blockwishlist for blacklist, and it's good.

File: src/PrestaShopBundle/Install/Install.php [hunk 1]
```diff
@@ -891,6 +891,7 @@ public function configureShop(array $data = [])
     public function getModulesList()
```
Comment (matks) [inline src/PrestaShopBundle/Install/Install.php:895]: Wah 😮  I did not know about this hardcoded list. It's a key item to be aware of though.

**Review by matks — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @Progi1984,

After a fresh install of the PR, the `blockwishlist, is well installed.
But the archived modules still exist on the Module Catalog page
The native archived modules must be removed & they must be not displayed on the Module Catalog page.

1. sekeywords  
image
2. statsequipment 
image
3. statsorigin 
image
4. statsvisits 
image


Thank you!

**Comment by Progi1984** [comment]
@khouloudbelguith I think the problem is due to addons cache. Please remove your cache and it should be ok.

**Comment by Progi1984** [comment]
@khouloudbelguith Addons API returns some archived modules : https://api.addons.prestashop.com/?version=1.7.8.0&iso_lang=en&iso_code=fr&format=json&method=listing&action=native

Issue created : https://forge.prestashop.com/browse/PLFS-3386

**Review by PierreRambaud — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @Progi1984,

It is ok :heavy_check_mark: 
https://drive.google.com/file/d/1P1ZlBwR4kbMKUB8HpOQ9sdkZ4o4-TpCT/view?usp=sharing

Thnk you!

**Comment by Progi1984** [comment]
Thanks @khouloudbelguith

**Pull request merged**